examples/index.html

```diff
@@ -38,7 +38,7 @@
   <!-- jspdf to enable save image as pdf -->
   <script src="../node_modules/jspdf/dist/jspdf.min.js" type="text/javascript" ></script>
 
-  <script src="lib/scrollToTop.js"></script>
+  <!-- <script src="lib/scrollToTop.js"></script> -->
```

Comment:
Does this thing not exist?

Reply:
Seems like it. Not sure what it does or where it might be. I commented out this link to clear up the error on the console.